exasol_script_languages_container_ci_setup/lib/aws/wrapper/cloudformation_service.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, Dict, Optional, Callable
+
+from exasol_script_languages_container_ci_setup.lib.aws.wrapper.datamodels.cloudformation import ValidationResult, \
+    ListStackResourcesResult, NextToken
+from exasol_script_languages_container_ci_setup.lib.aws.wrapper.datamodels.common import PhysicalResourceId
+
+
+class CloudFormationService:
+    def __init__(self, boto_client):
+        self._boto_client = boto_client
+
+    @property
+    def boto_client(self) -> Any:
```

Comment:
You are returning `Any` so is it relevant that it is named `boto_client` rather than just `client`?

Reply:
good point, currently this should be a thin wrapper around boto and we return that, because we still have some functionality which needs the boto client directly. Maybe in later stages, I can remove that.